### PR TITLE
Fixed conformance statements

### PR DESCRIPTION
**Pull request by brentzundel**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/442.html" title="Last updated on Mar 5, 2019, 3:17 PM UTC (dad8044)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/442/5f09c57...dad8044.html" title="Last updated on Mar 5, 2019, 3:17 PM UTC (dad8044)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1262,14 +1289,14 @@ <h3>Proofs (Signatures)</h3>
         <h3>Expiration</h3>
 
         <p>
-Expiration information for a <a>credential</a> MAY be provided by adding the
```
Comment (brentzundel) [inline index.html]: This language should be kept.

File: index.html [hunk 2]
```diff
@@ -1262,14 +1289,14 @@ <h3>Proofs (Signatures)</h3>
...
-following <a>property</a>:
+If expiration information for a <a>credential</a> is needed, then it MUST be provided
```
Comment (brentzundel) [inline index.html]: This is not testable.

File: index.html [hunk 3]
```diff
@@ -1302,15 +1329,15 @@ <h3>Expiration</h3>
         <h3>Status</h3>
 
         <p>
-Information about the current status of a <a>verifiable credential</a>, such
-as whether it is suspended or revoked, MAY be provided using the
+If information about the current status of a <a>verifiable credential</a>, such
+as whether it is suspended or revoked, is needed, then it MUST be provided using the
```
Comment (brentzundel) [inline index.html]: This is not testable.

File: index.html [hunk 4]
```diff
@@ -1302,15 +1329,15 @@ <h3>Expiration</h3>
         <h3>Status</h3>
 
         <p>
-Information about the current status of a <a>verifiable credential</a>, such
```
Comment (brentzundel) [inline index.html]: This language should be kept.

File: index.html [hunk 5]
```diff
@@ -1854,7 +1881,7 @@ <h3>Data Schemas</h3>
         </ul>
 
         <p>
-The expression of a data schema can be performed using the following
+The expression of a data schema MUST be performed using the following
```
Comment (brentzundel) [inline index.html]: This is not testable.

File: index.html [hunk 6]
```diff
@@ -1971,7 +1998,7 @@ <h3>Refreshing</h3>
 updates of the <a>credential</a>.
         </p>
         <p>
-A refresh service can be expressed using the following <a>property</a>:
+A refresh service MUST be expressed using the following <a>property</a>:
```
Comment (brentzundel) [inline index.html]: This is not testable.

File: index.html [hunk 7]
```diff
@@ -1981,8 +2008,9 @@ <h3>Refreshing</h3>
 more refresh services that provides enough information to the recipient's
 software such that the recipient can refresh the credential. Each
 <code>refreshService</code> value MUST specify its <code>type</code> (for
-example, <code>ManualRefreshService2018</code>), and its URL as the value of
-<code>id</code>. The precise content of each refresh service is determined by
```
Comment (brentzundel) [inline index.html]: This wording is more clear.

File: index.html [hunk 8]
```diff
@@ -2044,13 +2072,13 @@ <h3>Terms of Use</h3>
         </p>
 
         <p>
-Terms of use MAY be expressed using the following <a>property</a>:
+Terms of use MUST be expressed using the following <a>property</a>:
```
Comment (brentzundel) [inline index.html]: This is not testable.

File: index.html [hunk 9]
```diff
@@ -2176,16 +2204,18 @@ <h3>Evidence</h3>
         </p>
 
         <p>
-Evidence information can be expressed using the following <a>property</a>:
+Evidence information MUST be expressed using the following <a>property</a>:
```
Comment (brentzundel) [inline index.html]: This is not testable.

File: index.html [hunk 10]
```diff
@@ -2176,16 +2204,18 @@ <h3>Evidence</h3>
...
           <dt><var>evidence</var></dt>
           <dd>
 The value of the <code>evidence</code> <a>property</a> MUST be one or more
-evidence schemes providing enough information to a <a>verifier</a> to determine
-whether the evidence gathered by the <a>issuer</a> meets its requirements for
-issuing a <a>credential</a>. The precise content of each evidence scheme is
+evidence schemes providing enough information for a <a>verifier</a> to determine
+whether the evidence gathered by the <a>issuer</a> meets its confidence requirements for
+relying on the <a>credential</a>. Each evidence scheme is identified by its <a>type</a>.
+The  <code>id</code> property is optional, but if present, SHOULD contains a URL that
```
Comment (brentzundel) [inline index.html]: "SHOULD contain a URL"

**Comment by msporny** [comment]
Just a quick heads up that I cleaned this PR up quite a bit on the flight home from Barcelona, there were a number of things that were problematic with it (problematic changes from "can" to MAY, changes in formatting that were not in line w/ rest of document, new sections w/o examples, etc.). I need to also apply @brentzundel's comments in this PR. Once I get @brentzundel's comments merged, I suggest we close this PR and continue with #443 (because it is based on this PR, but with changes that I think will stick).

Please continue the discussion in https://github.com/w3c/vc-data-model/pull/443.

**Comment by brentzundel** [comment]
Sounds good to me.

**Comment by David-Chadwick** [comment]
Happy to continue with #443 so you can close this PR

**Comment by brentzundel** [comment]
I added comments to PR #443, so this PR may be closed.

**Pull request closed without merging**